packages/dotcom-ui-layout/browser.js

```diff
@@ -8,8 +8,7 @@ import oTypography from 'o-typography'
 import 'focus-visible'
 
 export function init({ headerOptions = {}, footerOptions = {} } = {}) {
-  const rootElement = document.querySelector('.n-layout')
-  oTypography.init(rootElement)
+  oTypography.init()
```

Comment:
As we're going to be pull this logic in from the base styles package, I'm pretty sure we can remove this line from the layout 🙂